README.md

```diff
@@ -41,6 +41,34 @@ git submodule update --init --recursive
 make run
 ```
 
+### Generating specific products
+
+In order to speed up build times, it's possible to generate a specific subset of products and their corresponding versions by specifying the `KONG_PRODUCTS` env variable. It takes a comma-separated list of products and for each product, the list of versions the versions to be generated separated by semi-colons, in the following way.
+
+```bash
+KONG_PRODUCTS=<product>:<version>;<version>,<product>:<version>;hub
+```
+
+For example, running
+
+```bash
+KONG_PRODUCTS=gateway:2.8.x;3.3.x,mesh:2.2.x,hub make run
```

Comment:
I can't get multiple versions for one product to work.

* This works:
    ```
    KONG_PRODUCTS=gateway:2.8.x,mesh:2.2.x make run
    ```

* This errors out:
    ```
    KONG_PRODUCTS=gateway:2.8.x;3.3.x,mesh:2.2.x make run
    ```
    
    Response:
    ```
    zsh: command not found: 3.3.x,mesh:2.2.x
    ```

* This seems like it works, but in the output there's only gateway 2.8.x and mesh 2.2.x:

    ```
    KONG_PRODUCTS=gateway:2.8.x:3.3.x,mesh:2.2.x make run
    ```

* This also seems like it works, but this time it builds only gateway 3.3.x and mesh 2.2.x:

    ```
    KONG_PRODUCTS=gateway:2.8.x,gateway:3.3.x,mesh:2.2.x make run
    ```